src/main/kotlin/de/itemis/mps/gradle/Common.kt

```diff
@@ -23,6 +27,7 @@ open class BasePluginExtensions {
     var macros: List<Macro> = emptyList()
     var projectLocation: File? = null
     var debug = false
+    var executable: Any? = null
```

Comment:
Why is this `Any`? What is this used for? 

If its a `File` pointing to some executable then it should be a `File` if its path as a string then `String`.

Reply:
The variable name, idea and type copy pasted from RunAntScript task. I just wanted to be consistent...